src/pages/find-offers/FindOffers.tsx

```
@@ -1,7 +1,59 @@
+import Loader from '~/components/loader/Loader'
+import { offerService } from '~/services/offer-service'
+import NotFoundResults from '~/components/not-found-results/NotFoundResults'
+import OfferRequestBlock from '~/containers/find-offer/offer-request-block/OfferRequestBlock'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { styles } from '~/pages/find-offers/FindOffer.styles'
 
 const FindOffers = () => {
-  return <PageWrapper>Find offers</PageWrapper>
-}
+  const { t } = useTranslation()
+  const [hasFetched, setHasFetched] = useState(false);
```

ESLint

[error] Delete `;`

(prettier/prettier)